cli/webhook.py

```diff
@@ -0,0 +1,125 @@
+import slackweb
+from sets import Set
+from datetime import datetime
+from slackclient import SlackClient
+
+from cli.settings import Settings
+from cli.appconfig import AppConfig
+
+
+class WebHook:
+
+    def __init__(self):
+        self.disabled = True
+        self.emoji = ':rocket:'
+        self.defChannel = ''
+        self.username = 'rogeros-deploy-bot'
+        self.settingObj = Settings()
+        self.appconfigObj = AppConfig()
+
+    def webhookSetting(self):
+        try:
+            config_dir = self.settingObj.getConfigDir()
+            roger_env = self.appconfigObj.getRogerEnv(config_dir)
+            if 'slack_webhook_url' in roger_env.keys():
+                self.webhookURL = roger_env['slack_webhook_url']
+            if 'slack_api_token' in roger_env.keys():
+                self.token = roger_env['slack_api_token']
+            if 'default_slack_channel' in roger_env.keys():
```

Comment:
minor - Consider renaming this to `slack_default_channel` to be consistent with the other vars.